app/views/payment-links/manage.njk

```diff
@@ -12,10 +12,26 @@
 {% block mainContent %}
 <section class="govuk-grid-column-two-thirds">
 {% if flash.createPaymentLinkSuccess %}
-    {{ successNotification({
-      heading: "Your payment link is now live",
-      body: "Give this link to your users to collect payments for your service."
-    }) }}
+  {% if isTestGateway %}
+    {% set heading = "Your test payment link is ready to use" %}
+    {% set body %}
+      <span>
+        To collect payments from your users with this link you will need to
```

Comment:
```suggestion
        To collect payments from your users you will need to  ?
```

Reply:
As there's no line breaking content in this span everything in the block will be treated as the same line, they're on separate lines here for readability as the link in particular is long

Reply:
I was trying to convey something using suggestion but clearly not done well :) . Do we need `with this link` as it won't be same link after live ?

Reply:
Ah! My bad, I thought the question mark was part of the suggestion 🤦

Reply:
Content has been updated.